effectly/src/hooks/Calendar/useCalendarData.tsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react";
+import { collection, getDocs } from "firebase/firestore";
+import { database } from "../../core/firebase/firebase";
+
+const useCalendarData = (userId: string) => {
+  const [calData, setCalData] = useState<any[]>([]);
+
+  const setCalendarData = async () => {
+    const dataQuery = collection(database, "users", `${userId}`, "calendar");
```

Comment:
Maybe would be better to create middleware file for the queries on database ? That would help us to divide queries for specific collections, and that would move the queries from components.

Reply:
Done with [0dd7d77](https://github.com/wooczak/effectly/pull/8/commits/0dd7d7753e9ad711b72b31cd3281432681307815)